packages/block-library/src/latest-comments/style.scss

```
@@ -21,22 +20,22 @@
 }
 
 .wp-block-latest-comments__comment-excerpt p {
-	font-size: 14px;
+	font-size: 0.875em;
 	line-height: 1.8;
-	margin: 5px 0 20px;
+	margin: 0.35em 0 1.4em;
```

This would be slightly closer to the original value.
```suggestion
	margin: 0.36em 0 1.4em;
```

---

You are right, fixing :+1: